redbot/cogs/admin/admin.py

```diff
@@ -272,7 +272,9 @@ async def editrole_colour(
             `[p]editrole colour Test #ff9900`
         """
         author = ctx.author
-        reason = "{}({}) changed the colour of role '{}'".format(author.name, author.id, role.name)
+        reason = _("{author} ({author.id}) changed the colour of role '{role.name}'").format(
```

Comment:
I'm pretty sure this (and similar) will error, because having named fillins requires kwargs in the format call.

Reply:
Of course, this is fixed now, thanks!